game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -6,7 +6,33 @@ package io.aigar.game
  * of the players.
  */
 class GameThread extends Runnable {
+  val MillisecondsPerTick = 16
+
+  private var _states: Map[Int, GameState] = Map()
```

Comment:
I thought I saw this somewhere, but can't find the source anymore. Removing the underscore.